packages/talos_certifier/src/services/certifier_service.rs

```diff
@@ -96,7 +98,12 @@ impl CertifierService {
         let conflict_candidate: Option<CandidateMessage> = self.get_conflict_candidates(&outcome);
 
         // Create the Decision Message
-        Ok(DecisionMessage::new(message, conflict_candidate, outcome, suffix_head))
+        let mut dm = DecisionMessage::new(message, conflict_candidate, outcome, suffix_head);
+        dm.can_received_at = message.received_at;
```

Comment:
could you look at this comment
https://github.com/kindredgroup/talos/pull/53/files#r1242768001